components/central-application-gateway/README.md

```diff
@@ -73,23 +72,37 @@ As a result, the Central Application Gateway:
    ```bash
    {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
 
+#### Standalone mode for Compass - simplified API
+
+The standalone mode can also be used for Compass bundles with a single API definition.
+This means that `{API_DEFINITION_NAME}` should be removed from the URL and its pattern looks like
+```
+{APPLICATION_NAME}/{API_BUNDLE_NAME}/{TARGET_API_PATH}
+```
+**Invocation of service bundles configured with multiple API definitions will result in 400 Bad Request failure.**
+
 ### Compass mode
-If **disableLegacyConnectivity** is `true`, the proxy API exposes the following endpoint:
+The proxy API exposes the following endpoint:
 ```bash
 {APPLICATION_NAME}/{API_BUNDLE_NAME}/{API_DEFINITION_NAME}/{TARGET_API_PATH}
-``` 
+```
 
 For instance, if the user registered the `cc-occ` API bundle with the `commerce-webservices` API definition in the `ec` application, they can send a request to the following URL:
 ```bash
 http://central-application-gateway:8082/ec/cc-occ/commerce-webservices/basesites
-``` 
+```
 
 As a result, the Central Application Gateway:
 1. Looks for the `cc-occ` service and the `commerce-webservices` entry in the `ec` Application CRD and extracts the target URL path along with the authentication configuration
 2. Modifies the request to include the authentication data
 3. Sends the request to the following path: 
    ```bash
    {TARGET_URL_EXTRACTED_FROM_APPLICATION_CRD}/basesites
+   ```
+
+#### Handling ambiguous API definition names
+A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CRD should be unique for a given application.
+Invocation of endpoints with duplicate names will result in **400 Bad Request** failure. In such a case one of the names should be changed to avoid ambiguity.
```

Comment:
```suggestion
#### Handling ambiguous API definition names

A combination of `{API_BUNDLE_NAME}` and `{API_DEFINITION_NAME}` which are extracted from an Application CRD should be unique for a given application.
Invocation of endpoints with duplicate names will result in a **400 Bad Request** failure. In such a case, one of the names should be changed to avoid ambiguity.
```